src/server/socketHandlers/wssConnectionHandler.js

```diff
@@ -12,8 +12,7 @@ import {TREBUCHET_WS} from '@mattkrick/trebuchet-client'
 
 const APP_VERSION = packageJSON.version
 export default function connectionHandler (sharedDataLoader, rateLimiter) {
-  return async function socketConnectionHandler (socket) {
-    const req = socket.upgradeReq
+  return async function socketConnectionHandler (socket, req) {
```

Comment:
websockets no longer hold onto the req. in the future, we could clear this for ~20% memory savingss